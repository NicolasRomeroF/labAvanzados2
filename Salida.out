Se requieren 1052 movimientos
