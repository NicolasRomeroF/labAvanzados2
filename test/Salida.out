Se requieren 2 movimientos
